x-pack/plugins/cloud_security_posture/public/components/translations.ts

```diff
@@ -40,3 +40,33 @@ export const CSP_EVALUATION_BADGE_PASSED = i18n.translate(
     defaultMessage: 'PASSED',
   }
 );
+
+export const ERROR_LOADING_DATA = i18n.translate('xpack.csp.pageTemplate.loadErrorMessage', {
+  defaultMessage: "We couldn't fetch your cloud security posture data",
+});
+
+export const NO_DATA_CONFIG_TITLE = i18n.translate('xpack.csp.pageTemplate.noDataConfigTitle', {
+  defaultMessage: 'Understand your cloud security posture',
+});
+
+export const NO_DATA_CONFIG_SOLUTION_NAME = i18n.translate(
+  'xpack.csp.pageTemplate.noDataConfig.solutionName',
```

Comment:
`Name` is not mentioned in the i18n guidelines as a valid `message id`. try using something from the list mentioned here: https://github.com/elastic/kibana/blob/main/packages/kbn-i18n/GUIDELINE.md
try to match it to the element you are rendering as mentioned above